imgui/core.pyx

```diff
@@ -1646,6 +1708,29 @@ def label_text(str label, str text):
     cimgui.LabelText(_bytes(label), "%s", _bytes(text))
 
 
+def text_unformatted(str text):
+    """Doesn't require ``NULL`` terminated string if 'text_end' is specified.
```

Comment:
Since we don't allow passing the `text_end` argument this docstring is not valid.